modules/examiner/test/ExaminerTest.php

```diff
@@ -22,11 +23,53 @@
  * @category Test
  * @package  Loris
  * @author   Tara Campbell <tara.campbell@mail.mcgill.ca>
+ * @author   Wang Shen <wangshen.mcin@gmail.com>
  * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
  * @link     https://github.com/aces/Loris
  */
 class ExaminerTest extends LorisIntegrationTest
 {
+    /**
+    * Insert testing data
+    *
+    * @return void
+    */
+    public function setUp()
+    {
+        parent::setUp();
+         $window = new WebDriverWindow($this->webDriver);
+         $size   = new WebDriverDimension(1024, 1768);
+         $window->setSize($size);
+        $this->DB->insert(
+            "psc",
+            array(
+             'CenterID'   => '9999999',
+             'Name'       => 'TEST_Site',
+             'Study_site' => 'Y',
+             'StateID'    => '0',
+             'Alias'      => 'DDD',
+             'MRI_alias'  => 'TESTTEST',
```

Comment:
Should there be a psc site setup in the basic LorisIntegrationTest class instead of duplicating this for all the different test setup/teardown functions?